Gruntfile.js

```diff
@@ -144,6 +157,13 @@ module.exports = function(grunt) {
       },
       stop_redis: {
         command: 'pgrep redis-server | xargs kill'
+      },
+
+      emojify: {
+        command: 'rsync -az bower_components/emoji-parser/emoji public/images && echo "var base_emojis = [" > public/javascripts/base_emoji.js && ls -1 public/images/emoji | grep ".png" | sed -e \'s/\\..*$//\' | xargs -I{} echo \\\"{}\\\" | tr \"\\n\" "," | sed "s/,$//" >> public/javascripts/base_emoji.js && echo "];" >> public/javascripts/base_emoji.js',
```

Comment:
Yeah I'm thinking there's probably a way to use Node's fs module to list the files, and then just `JSON.stringify` the thing.  But I'm also fine with this for now.